frontend/src/lib/util.ts

```diff
@@ -160,7 +160,7 @@ export function useFilterFunc<
   const filter = useTypedSelector(state => state.filter);
 
   return (item: T, search?: string) => {
-    if (!!item.metadata) {
+    if (item && item.metadata) {
```

Comment:
`if(item?.metadata) {`
would be a bit simpler

Reply:
Thanks. Fixed.